apps/api/src/database/schema.d.ts

```diff
@@ -53,6 +53,10 @@ export interface ServiceUser extends schema.ServiceUser {
 	address?: schema.Address;
 }
 
+export interface Subscription extends schema.Subscription {
+	serviceUser: ServiceUser;
+}
```

Comment:
I tried to make this change but it causes errors in the code because the `serviceUser` property doesn't include `address` by default (which we make use of), whereas in this current type declaration we're using the custom defined `ServiceUser` type that includes the address.

Reply:
Ah, you can include that too then:

`SubscriptionGetPayload<{include: {serviceUser: {include: {address: true}}>`

Reply:
Ah cool, that's done.